### ISSUE TITLE
Add touch shortcuts for undo and redo

### ISSUE THREAD

**Issue by undeadf0x** [tldraw/tldraw#1336]
### What's the feature?

Something that would be nice for tldraw is some "touch shortcuts" for undoing and redoing

Tapping with two fingers would undo, three would redo, this would make using tldraw on touchscreen a lot easier

examples: ibisPaint X is the only software i can think of that does this, https://ibispaint.com/?lang=en-US

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by steveruizok** [comment]
This exists in Procreate too. I actually started working on this a few months ago, however I had to step back because of how complex the feature was. Specifically, we needed a way to "fork" our undo/redo history.

With a pencil tool enabled, the first touch would created a new draw shape, but then the second touch would trigger the "double tap" gesture. We would need to first undo the drawing, then undo again to actually commit the undo.

That's fine if we're at the end of the stack, however repeating the gesture to undo again would remove our redo stack when that first shape is created. This bug also occurs when redoing.

The solution was to rethink how we handled touch events, where a pointer down might be a pointer down or double pointer down or triple pointer down. We'll need to create a better gesture recognizer that takes into account the type of event (e.g. touch vs. mouse) in order to support this type of feature.
